components/navbar/camera.js

```diff
@@ -40,7 +40,7 @@ export default class CameraComponent extends React.Component {
     }
     // Camera Permission
     const { status } = await Permissions.askAsync(Permissions.CAMERA);
-    this.setState({ hasPermission: status === 'granted' });
+    this.setState({ hasPermission: status === 'granted' })
```

Comment:
ah previously the lack of semi colon was giving me an error but if it did not for you then all good